### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by david-kow**
Two changes:
- bump alpha to beta in readme,
- add link to Elastic forums (we have that in issue template, but this will be more visible and could get people to the right place quicker, see https://github.com/elastic/cloud-on-k8s/issues/1835).

### CODE REVIEW

**Comment by david-kow** [comment]
jenkins test this please

**Comment by pebrc** [comment]
Should we merge this post-release?

File: README.md [hunk 1]
```diff
@@ -25,3 +25,5 @@ Supported versions:
 Check the [Quickstart](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-quickstart.html) if you want to deploy you first cluster with ECK.
 
 If you want to contribute to the project, check our [contributing guide](CONTRIBUTING.md) and see [how to setup a local development environment](dev-setup.md).
+
+For general questions, please see Elastic [forums](https://discuss.elastic.co/c/eck).
```
Comment (anyasabo) [inline README.md]:
```suggestion
For general questions, please see the Elastic [forums](https://discuss.elastic.co/c/eck).
```

**Comment by david-kow** [comment]
> Should we merge this post-release?

We do have references to beta right now on master, but since main readme is more visible we can wait.

**Comment by alaudazzi** [comment]
Fixes #1890

**Pull request merged**